netbox/info.json

```diff
@@ -0,0 +1,2668 @@
+      "category": "investigation",
+      "annotation": "update_ip_address",
+      "enabled": true,
+      "output_schema": {
+        "id": "",
+        "url": "",
+        "display": "",
+        "family": {
+          "value": "",
+          "label": ""
+        },
+        "address": "",
+        "vrf": {
+          "id": "",
+          "url": "",
+          "display": "",
+          "name": "",
+          "rd": ""
+        },
+        "tenant": {
+          "id": "",
+          "url": "",
+          "display": "",
+          "name": "",
+          "slug": ""
+        },
+        "status": {
+          "value": "",
+          "label": ""
+        },
+        "role": {
+          "value": "",
+          "label": ""
+        },
+        "assigned_object_type": "",
+        "assigned_object_id": "",
+        "assigned_object": "",
+        "nat_inside": {
+          "id": "",
+          "url": "",
+          "display": "",
+          "family": "",
+          "address": ""
+        },
+        "nat_outside": [
+          {
+            "id": "",
+            "url": "",
+            "display": "",
+            "family": "",
+            "address": ""
+          }
+        ],
+        "dns_name": "",
+        "description": "",
+        "comments": "",
+        "tags": [
+          {
+            "id": "",
+            "url": "",
+            "display": "",
+            "name": "",
+            "slug": "",
+            "color": ""
+          }
+        ],
+        "custom_fields": {
+          "additionalProp1": "",
+          "additionalProp2": "",
+          "additionalProp3": ""
+        },
+        "created": "",
+        "last_updated": ""
+      },
+      "parameters": [
+        {
+          "title": "IP Address ID",
+          "name": "id",
+          "visible": true,
+          "required": true,
+          "editable": true,
+          "type": "integer",
+          "tooltip": "Specify the unique integer value identifying this IP address in Netbox instance.",
+          "description": "Specify the unique integer value identifying this IP address in Netbox instance."
+        },
+        {
+          "title": "Fields to be updated",
+          "name": "patch_fields",
+          "visible": true,
+          "required": true,
+          "editable": true,
+          "type": "json",
+          "value": {
+            "address": "string",
+            "vrf": 0,
+            "tenant": 0,
+            "status": "active",
+            "role": "loopback",
+            "assigned_object_type": "string",
+            "assigned_object_id": 9223372036854776000,
+            "nat_inside": 0,
+            "dns_name": "*.G7tafc4lbh4wzM6rk8u4RVrwSZQjKHUkopm6odkzY7ao3A5D6ZBtPlhdsXOVwGSq3H9vuWbb.D5EWQI0yFjpnkXiBKJpSObjpNovFX8-.F4VlX_dXq0QVuGYb6vCfdm.fn8ByqgCQBiD44X2qBoT6bQ_0ziYqaTedzFD4UK9lxhrh9ay_m0cHf9u5FYiUjRkEMIEcSEPRvlp.KR8hmwgztO7ELv-b.W4uo1ZOWY99FKQfaztOyjuYb2O6j4OE",
```

Comment:
Please remove this example or keep keys only with empty values.
Provide actual example in sample playbook